pkg/cache/sql/informer/listoption_indexer.go

```diff
@@ -51,8 +55,21 @@ const (
 	createFieldsIndexFmt = `CREATE INDEX "%s_%s_index" ON "%s_fields"("%s")`
 
 	failedToGetFromSliceFmt = "[listoption indexer] failed to get subfield [%s] from slice items: %w"
+
+	// 'key' contains values from the column key of table nodes
+	createLabelsTableFmt = `CREATE TABLE IF NOT EXISTS "%s_labels" (
+		key TEXT NOT NULL REFERENCES "%s"(key) ON DELETE CASCADE,
+		label TEXT NOT NULL,
+		value TEXT NOT NULL,
+		PRIMARY KEY (key, label)
+	)`
```

Comment:
We will need at least an `INDEX` over `label` here, to speed up queries at retrieval time.

Reply:
I added that today.